### PR TITLE
Greyscale wirecutters

### PR DESCRIPTION
**Pull request by ghost**
Greyscales wirecutters.

:cl: More Robust Than You
imageadd: Wirecutters can come in even more colors now!
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
cutters_cutty_thingy

**Comment by MoreRobustThanYou** [comment]
screwdriver_screwybits

**Comment by ShizCalev** [comment]
Will made #31871 involve wirecutters as well, for the record.

**Comment by Bawhoppen** [comment]
:+1: x :100:
CANNOT THUMBS UP THIS ENOUGH

File: code/game/objects/items/tools/wirecutters.dm [hunk 1]
```diff
@@ -20,14 +21,34 @@
 	toolspeed = 1
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 30)
 	var/random_color = TRUE
+	var/static/list/wirecutter_colors = list(
+		"blue" = rgb(24, 97, 213),
+		"red" = rgb(149, 23, 16),
+		"pink" = rgb(213, 24, 141),
+		"brown" = rgb(160, 82, 18),
```
Comment (praisenarsie) [inline code/game/objects/items/tools/wirecutters.dm]: my favourite color

**Comment by ShizCalev** [comment]
Ongoing freeze, #32586. Feel free to reopen once it's over.

**Comment by JJRcop** [comment]
@ShizCalev If maintainers close a PR then OP can't reopen, only maintainers can

**Pull request closed without merging**